go/interpreter/lfvm/gas.go

```diff
@@ -410,7 +401,17 @@ func gasSStoreEIP2929(c *context) (tosca.Gas, error) {
 		// If the caller cannot afford the cost, this change will be rolled back
 		c.context.AccessStorage(c.params.Recipient, slot)
 	}
-	value := tosca.Word(y.Bytes32())
+
+	////////////////////////////////////////////////////////////////////////
+	// If we fail the minimum gas availability invariant, fail (0)
+	if c.gas <= SstoreSentryGasEIP2200 {
+		c.status = statusOutOfGas
+		return 0, errNotEnoughGasReentrancy
+	}
+
+	// Gas sentry honoured, do the actual gas calculation based on the stored value
+	zero := tosca.Word{}
```

Comment:
zero can still be declared later on.

Reply:
changes in this function reverted, since they had no effect and were part of a failed experiment.